packages/plugins/test/mocha/cli/deploy.spec.ts

```diff
@@ -240,14 +242,6 @@ describe('Deploy CLI command', () => {
       ])
       const executionMode = ExecutionMode.LiveNetworkCLI
 
-      // Override the `isLiveNetwork` function to always return `true`.
-      context.isLiveNetwork = async (
-        // eslint-disable-next-line @typescript-eslint/no-unused-vars
-        provider_: SphinxJsonRpcProvider | HardhatEthersProvider
-      ): Promise<boolean> => {
-        return true
-      }
-
```

Comment:
No longer necessary because we mock the `isLiveNetwork` on the SphinxContext so that it always returns `true`